src/universal/components/MeetingHelp/HelpMenuToggle.js

```diff
@@ -0,0 +1,23 @@
+import IconLabel from 'universal/components/IconLabel'
+import React from 'react'
+import styled from 'react-emotion'
+import RaisedButton from 'universal/components/RaisedButton'
+import {meetingHelpWithBottomBar} from 'universal/styles/meeting'
+
+const StyledButton = styled(RaisedButton)(({isFacilitating}) => ({
+  bottom: isFacilitating ? meetingHelpWithBottomBar : '1.25rem',
```

Comment:
previously, this was part of `MeetingHelpMenuLayout` and `StyledMeetingHelpMenuLayout`. Since we're styling the toggle, it's more expressive to put those styles on the toggle directly.